### DISCUSSION TITLE
Is it possible to change the ScrollHideDelay value in a Prism component?

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#2240] [Q&A] [answered]
### What package has an issue

@mantine/core

### Describe the bug

It seems that the Prism component is using the ScrollArea component as the default component by default. So, how can I change the ScrollHideDelay value of the ScrollArea component?

### What version of @mantine/hooks page do you have in package.json?

5.2.3

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/dry-pine-b0b7n6?file=/src/App.tsx

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by rtivital** [comment, marked as answer]
Yes, you can provide your own `scrollAreaComponent` prop: `<Prism scrollAreaComponent={(props) => <ScrollArea {...props} scrollHideDelay={8000} />} />`

**Reply by ryuhangyeong** [reply]
Thanks you~
